ext/serializer.c

```diff
@@ -828,6 +856,33 @@ static void _serialize_meta(zval *el, ddtrace_span_data *span) {
         add_assoc_str(meta, "_dd.span_links", buf.s);
     }
 
+    zend_array *peer_service_sources_zv = ddtrace_spandata_property_peer_service_sources(span);
```

Comment:
can you please not use a _zv suffix for something not a zval? That's confusing :-D